### PR TITLE
Add Chrome 60

### PR DESCRIPTION
**Pull request by sscotth**

### CODE REVIEW

File: data-esnext.js [hunk 1]
```diff
@@ -1179,6 +1179,7 @@ exports.tests = [
         babel: true,
         typescript: typescript.corejs,
         duktape20: false,
+        chrome60: true,
```
Comment (chicoxyzzy) [inline data-esnext.js]: That's not true for Canary 60.0.3077.0

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]

File: data-esnext.js (hunk 1 above)
Comment (sscotth) [reply data-esnext.js]: Are you sure?

screen shot 2017-04-21 at 3 44 44 am

Comment (chicoxyzzy) [reply data-esnext.js]: Yes I'm pretty sure.
screen shot 2017-04-21 at 10 46 53

Canary has A/B testing as was discussed before in older issues and PRs so maybe you are just the one of those lucky testers :)

Comment (sscotth) [reply data-esnext.js]: Interesting. I actually tried it on two (macOS) computers. I'll remove it.

**Comment by afmenez** [comment]
How about retiring some old versions when a new one is added?  Chrome 50 was released more than a year ago, I believe anything older than that should be tagged with "obsolete:very" ...

**Comment by sscotth** [comment]
That sounds more appropriate for a separate issue/pull request. It could even be automated in the build script with proper retire dates.

**Review by chicoxyzzy — APPROVED** [review]
I am ready to merge this after resolving conflicts 👍

**Comment by chicoxyzzy** [comment]
Thank you @sscotth!

**Pull request merged**